client/src/App.js

```diff
@@ -49,14 +49,41 @@ export default () => {
     signupDisclosure.onClose();
 
     // TODO: Make fetch() request here to signup
-
-    loginDisclosure.onOpen();
+    fetch('http://localhost:5000/signup', {
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        'Content-type': 'application/json'
+      },
+      body: JSON.stringify(values)
+    })
+    .then(function (data) {
```

Comment:
Likewise, using Promises in JS is kinda ugly. It gets really complicated. You're in a really awesome Create React App where all your code is transpiled for you: write ES6/7. I'd suggest you start using `async/await` instead of promises:

- https://www.youtube.com/watch?v=vn3tm0quoqE
- https://levelup.gitconnected.com/async-await-vs-promises-4fe98d11038f